README.md

```diff
@@ -1,2 +1,144 @@
-# js-libp2p-hop-relay-server
-A out of the box libp2p relay server with HOP
+# js-libp2p-hop-relay-server <!-- omit in toc -->
```

Comment:
Recommend renaming this to remove the redundancy of hop/server. Will need to be replaced in multiple places
```suggestion
# js-libp2p-relay-server <!-- omit in toc -->
```